### PR TITLE
Bump to GMT 6.2.0

### LINKED ISSUE #190: Update the links for GMT 6.2.0
GMT 6.2.0 已经发布，科大镜像 http://mirrors.ustc.edu.cn/gmt/ 也已经顺利同步。

需要将软件下载链接更新到 6.2.0 版本。

### PR DESCRIPTION
**Pull request by seisman**
Closes #190.

### CODE REVIEW

File: content/home/download.md [hunk 1]
```diff
@@ -68,19 +68,20 @@ subtitle = ""
...
 **GMT源码**:
-[v6.1.1](http://mirrors.ustc.edu.cn/gmt/gmt-6.1.1-src.tar.gz)
+[v6.2.0](http://mirrors.ustc.edu.cn/gmt/gmt-6.2.0-src.tar.gz)
 <br>
 **GSHHG数据**:
 [v2.3.7](http://mirrors.ustc.edu.cn/gmt/gshhg-gmt-2.3.7.tar.gz)
 <br>
 **DCW数据**:
-[v1.1.4](http://mirrors.ustc.edu.cn/gmt/dcw-gmt-1.1.4.tar.gz)
+[v2.0.0](http://mirrors.ustc.edu.cn/gmt/dcw-gmt-2.0.0.tar.gz)
```
Comment (core-man) [inline content/home/download.md:77]: 404 Not Found

Comment (seisman) [inline content/home/download.md:77]:
```suggestion
[v2.0.0](https://github.com/GenericMappingTools/dcw-gmt/releases/download/2.0.0/dcw-gmt-2.0.0.tar.gz)
```

Comment (core-man) [reply content/home/download.md:77]: Why does ustc mirror not sync dcw2.0.0？

Comment (core-man) [reply content/home/download.md]: How about gshhg?

Comment (seisman) [reply content/home/download.md]: The idea is to fully remove the FTP server. Future GSHHG versions also won't be uploaded to the FTP.

Comment (core-man) [reply content/home/download.md]: Shall we change it too?

Comment (seisman) [reply content/home/download.md]: GSHHG v2.3.7 is still available on the USTC mirror. It's definitely faster than github.

**Review by core-man — APPROVED** [review]

**Pull request merged**